projects/client-api/src/index.js

```diff
@@ -1670,6 +1671,48 @@ export function subscribeLabels({ onChange, onExit, onError, g }) {
         .subscribe({ error: onError });
 }
 
+/**
+ * Subscribe to graph simulation completion event
+ * @function subscribeLabels
+ * @param {@link GraphistryState} [g] A {@link GraphistryState} {@link Observable} or depricated, cache an object.
+ * @return {Subscription} A {@link Subscription} that can be used to react to the play updates
+ * @example
+ * GraphistryJS(document.getElementById('viz'))
+ *     .pipe(
+ *          map(playUpdates),
+ *          tap(() => console.log('Play completed')),
+ *     })
+ *     .subscribe();
+ */
+export function playUpdates(g) {
+    if (!(g.subscriptionAPIVersion >= 1)) {
+        return throwError(() => new Error('playUpdates is not available the currently embedded graphistry viz.'));
+    }
+
+    const selectionPath = ".labels";
```

Comment:
Out of curiosity, do we want to use `.labels` or `.selection.labels` here? Or are we going to create a new path for simulations to handle play updates?